vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -44,15 +44,15 @@ def __init__(
         self.optimize_streaming_latency = synthesizer_config.optimize_streaming_latency
         self.words_per_minute = 150
 
-    @tracer.start_as_current_span(
-        "synthesis", Context(synthesizer=SynthesizerType.ELEVEN_LABS.value)
-    )
     async def create_speech(
         self,
         message: BaseMessage,
         chunk_size: int,
         bot_sentiment: Optional[BotSentiment] = None,
     ) -> SynthesisResult:
+        create_speech_span = tracer.start_span(
```

Comment:
nit: technically we should start this right before we hit the endpoint no? (not that it'll change the time by much); it's also in other synthesizers

Reply:
Yes, it should be done the way you suggest. I'll change it. I left it at the top of the function to make it as similar as possible to the previous implementation.